src/Bicep.Core/Intermediate/ExpressionRewriteVisitor.cs

```diff
@@ -656,6 +656,12 @@ public virtual Expression ReplaceWildcardImportVariablePropertyReferenceExpressi
         return expression;
     }
 
+    void IExpressionVisitor.VisitResourceDerivedTypeExpression(ResourceDerivedTypeExpression expression) => ReplaceCurrent(expression, ReplaceResourceDerivedTypeExpression);
+    public virtual Expression ReplaceResourceDerivedTypeExpression(ResourceDerivedTypeExpression expression)
+    {
+        return expression;
```

Comment:
Missing implementation here, although it looks like that's the case for all of the import type functionality. Probably not a huge deal currently as I don't think we have any functionality which leverages rewriting for types - feel free to create tech debt issue if you don't want to address this now.

Reply:
The implementation for type expressions in this file should be complete... most of them (like `ResourceDerivedTypeExpression`) don't have any nested IR expressions. What should I add here?

Reply:
Oh my bad - I just assumed it would have child expressions.